### ISSUE TITLE
productMPS not working for N=1 spin (siteinds also breaks with N=1)

### ISSUE THREAD

**Issue by GTorlai** [ITensor/ITensors.jl#379]
Code:
```julia
sites = Index(2,tags="Site,n=1")
psi = productMPS(sites,1)
```
throws error `ERROR: MethodError: no method matching length(::Index{Int64})`.

By creating manually the MPS above, the functions `siteind(psi,1)` and `siteinds(psi)` also fail:
`ERROR: BoundsError: attempt to access 1-element Array{ITensor,1} at index [2]`

**Comment by emstoudenmire** [comment]
Thanks for pointing this out!

**Comment by GTorlai** [comment]
great, thanks!

**Comment by emstoudenmire** [comment]
Hi Giacomo, thanks again - just fixed this. Two things though:

1. Matt is working on some other fixes to go into a version 0.1.1 to be out today or tomorrow, so you won’t get the changes right away unless you start asking Pkg to track the master branch
2. the notation you were trying for productMPS wasn’t quite right. Instead you should call it like this:
```
site = Index(2,”Site,n=1”)
psi = productMPS([site],[1])
```
The main point above is that productMPS expects both inputs to be Vectors, either of Index objects (first input) or integers (second input)

**Comment by emstoudenmire** [comment]
However, there was a bug for N==1 even when using the correct code / input as above, so it was a helpful thing for us to know about

**Comment by GTorlai** [comment]
Indeed I tried both ways. Thanks so much for the quick fix!

**Comment by emstoudenmire** [comment]
Fixed by #380
